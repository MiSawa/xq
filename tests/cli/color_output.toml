args = [ "-C" ]

stdin = '''
1
"a"
["a", "b"]
'''

# stdout in .stdout file
