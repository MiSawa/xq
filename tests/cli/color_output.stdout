1
[32m"[0m[32ma[0m[32m"[0m
[1m[[0m
  [32m"[0m[32ma[0m[32m"[0m,
  [32m"[0m[32mb[0m[32m"[0m[1m
][0m
